crates/ursa-application/src/bin/genesis_committee.toml

```diff
@@ -0,0 +1,32 @@
+[[committee]]
+    owner = "0xCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC"
```

Comment:
Are these test committees?

Reply:
Could we move `https://github.com/qti3e/ursa-nodes` under the repo

Reply:
We could for sure, but its a very manual way of running a whole committee and not very maintainable. I think we ideally need something like this that facebook did in their implementation for running a whole committee https://github.com/facebookresearch/narwhal/tree/main/benchmark
Building something like that is on my roadmap- E2E Benchmarks/simulations. 
@qti3e thoughts on bringing in your repo?

Reply:
I think if it will be used then it should be moved over. If alternatives exist like the ones mentioned, that makes it better too